Makefile

```diff
@@ -25,29 +34,32 @@ build_wallet:
 ui_tests:
 	echo "UI Tests disabled"
 
-unit_tests:
```

Comment:
why is it removed?

Reply:
My bad I thought it was the same thing as integration_test. I will put it back.